ui/package.json

```diff
@@ -5,10 +5,12 @@
     "dependencies": {
         "@novnc/novnc": "^1.1.0",
         "ansi-256-colors": "^1.1.0",
+        "build": "^0.1.4",
```

Comment:
what's it for?

Reply:
it may be a bad merge.. i'll remove.
thanks